README.md

```diff
@@ -1,6 +1,103 @@
 # sc-stablecoin-rs
-Smart contracts for an over-collateralized stablecoin
 
-Original Lending/Liquidity components can be found here: https://github.com/ElrondNetwork/elrond-lend-rs/tree/main
+## Introduction
 
-Some of the code is duplicated in this repo and further adapted where needed.
+Stablecoins are a type of cryptocurrency that aims to have a 1:1 exchange rate between 1 crypto coin and 1 fiat equivalent (for example, having one coin have the same value as one dollar). As their name suggests, stablecoins aim to provide stability in the cryptocurrency world. Cryptocurrencies like Bitcoin can’t really be used as a day to day currency, as they're way too volatile. Fluctuations of 1-2k on a daily basis are just not acceptable for a currency.  
+
+There are two main types of stablecoins:
+- algorithmic
+- collateralized
+
+We're going to focus on the latter one in this document. This type of stablecoin is backed up by some item of equal value, meaning that you can always exchange back and forth with (almost) no price fluctuation. There are three main types of items being used as collateral:
+- Fiat currency (US Dollar)
+- Commodities (Interchangeable assets like jewelry or gold)
+- Other crypto coins
+
+In this case, we're focusing on the third one: cryptocoin-based collateral.  
+
+## Overcollateralized stablecoins
+
+We've talked a bit about collateralized stablecoins, but unfortunately, they've got a problem: it's very easy to get "free money" out of them. Their core mechanic is that you sell some sort of asset for its value in dollars at that point in time, and you can re-buy your asset back at a later time.  
```

Comment:
it's not very clear what you mean by "free money" and later buy-back.

Reply:
I'm not sure how to explain it better. I mean the case in which you sell your asset for a high value and later buy it back for a lower value.

Reply:
I still don't think it qualifies as free-money. The value should stay the same, what changes is the redemption rate.